src/views/Prescriptions/Entity/PatientCard/index.tsx

```diff
@@ -18,7 +18,7 @@ const PatientCard = ({ prescription, loading }: OwnProps) => {
       <ParagraphLoader loading={loading} paragraph={{ rows: 6 }}>
         <PatientContent
           patient={patient}
-          isPrenatal={prescription?.category[0]?.text === 'Prenatal'}
+          isPrenatal={prescription?.category && prescription?.category[0]?.text === 'Prenatal'}
```

Comment:
```suggestion
          isPrenatal={prescription?.category && prescription.category[0]?.text === 'Prenatal'}
```
Tu peux retirer le ? du deuxieme prescription comme tu l'as déjà testé dans la première condition.

Reply:
done :+1: